docs/parameters.yaml

```diff
@@ -525,6 +510,21 @@ type: string
 default: none
 components: ["origin", "director", "nsregistry"]
 ---
+name: Server.UIActivationCodeFile
+description: >-
+  If the origin's web user interface has not yet been configured, this file will
+  contain the activation code necessary to turn it on.
+type: string
+default: $ConfigBase/server-web-activation-code
+components: ["origin", "cache", "nsregistry", "director"]
+---
+name: Server.UIPasswordFile
```

Comment:
They will share the password and the one-time code. I think the password-sharing is intended if you spin up multiple services in the single process, but the activation code might have racing condition if you spin up multiple services at the start, at the same time, but I can't think of a solution on top of my head.

Reply:
Maybe make the `server-web-activation-code` something more server type specific? This might be something we decide to fix later.

Reply:
Agree. We can add prefix to the activation code so that each server can have independent activation code at the start, but this requires #280 to be finished first.